### DISCUSSION TITLE
Crashes on Windows 10

### DISCUSSION THREAD

**Discussion by RT-XD** [DesktopECHO/Pi-Hole-for-WSL1#111] [General]
Hi, 

I installed and have been trying this for a few days on windows 10. 
The task seems to have crashed on its own randomly a couple of times now, and I have to manually relaunch it.
Is there anywhere I can view logs to maybe investigate why/how this is happening?

Thanks :)

**Comment by DesktopECHO** [comment]
Hi, did you set-up the scheduled task so it run automatically when your computer starts up?

**Reply by RT-XD** [reply]
Yes I set up through task scheduler as per the instructions.
